codemeta.json

```diff
@@ -0,0 +1,83 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "license": "https://spdx.org/licenses/GPL-3.0",
+    "codeRepository": "https://github.com/javierrico/gLike",
+    "dateCreated": "2018-11-13",
+    "datePublished": "2020-07-30",
+    "dateModified": "2021-03-12",
+    "downloadUrl": "https://github.com/javierrico/gLike/releases/tag/v00.09.03",
+    "issueTracker": "https://github.com/javierrico/gLike/issues",
+    "name": "gLike",
+    "version": "v00.09.03",
+    "identifier": "10.5281/zenodo.3967385",
+    "description": "gLike is a general-purpose ROOT-based code framework for the numerical maximization of joint likelihood functions. The joint likelihood function has one free parameter (named g) and as many nuisance parameters as wanted, which will be profiled in the maximization process.",
```

Comment:
Is it possible to add the current description at the Zenodo page which is a bit longer than this one?